gatsby-config.js

```diff
@@ -23,37 +24,51 @@ module.exports = {
     )
   },
   plugins: flatten([
-    process.env.GATSBY_ACTIVE_ENV !== 'docs' ? [
-      {
-        resolve: `gatsby-plugin-page-creator`,
-        options: {
-          path: `${__dirname}/content/pages`,
-        },
-      },
-      {
-        resolve: `gatsby-source-wordpress`,
-        options: {
-          baseUrl: 'sparkpost.com',
-          protocol: 'https',
-          hostingWPCOM: false,
-          useACF: false,
-          excludedRoutes: [ '**/oembed/**', '**/akismet/**', '**/yoast/**' , '**/users/**', '**/settings', '**/pages', '**/yst_prominent_words', '**/comments', '**/statuses', '**/media' ]
-        }
-      }
-    ] : [],
-    process.env.GATSBY_ACTIVE_ENV === 'publish' ? [
-      {
-        resolve: `gatsby-plugin-algolia`,
-        options: {
-          appId: 'SFXAWCYDV8',
-          apiKey: process.env.ALGOLIA_TOKEN,
-          queries: [
-            require('./algolia/apiReference'),
-            require('./algolia/momentum')
-          ]
-        }
-      }
-    ] : [],
+    process.env.GATSBY_ACTIVE_ENV !== 'docs'
+      ? [
+          {
+            resolve: `gatsby-plugin-page-creator`,
+            options: {
+              path: `${__dirname}/content/pages`,
+            },
+          },
+          {
+            resolve: `gatsby-source-wordpress`,
+            options: {
+              baseUrl: 'sparkpost.com',
+              includedRoutes: [
```

Comment:
This is the only configuration that changed.  Instead of keeping track of everything that we need to exclude, I changed this to only fetch what we need.